test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1659,7 +1659,7 @@ UCS_TEST_SKIP_COND_P(test_ucp_wireup_asymmetric_ib, different_pci_bw_connect,
 {
     /* Enable cross-dev connection */
     /* coverity[tainted_string_argument] */
-    ucs::scoped_setenv path_mtu_env("UCX_RC_PATH_MTU", "1024");
+    ucs::scoped_setenv path_mtu_env("UCX_IB_PATH_MTU", "1024");
```

Comment:
can we ignore irrelevant things like path MTU when trying to understand if the remote device is on the same machine?

Reply:
why isn't it relevant?